client/project.go

```diff
@@ -225,13 +225,15 @@ type ResourceConfigResponse struct {
 	FunctionDefaultTimeout    *int64  `json:"functionDefaultTimeout"`
 	Fluid                     bool    `json:"fluid"`
 	ElasticConcurrencyEnabled bool    `json:"elasticConcurrencyEnabled"`
+	BuildMachineType          *string `json:"buildMachineType"`
 }
 
 type ResourceConfig struct {
 	FunctionDefaultMemoryType *string `json:"functionDefaultMemoryType,omitempty"`
 	FunctionDefaultTimeout    *int64  `json:"functionDefaultTimeout,omitempty"`
 	Fluid                     *bool   `json:"fluid,omitempty"`
 	ElasticConcurrencyEnabled *bool   `json:"elasticConcurrencyEnabled,omitempty"`
+	BuildMachineType          *string `json:"buildMachineType,omitempty"`
```

Comment:
`omitempty` here means we'll never actually send `null` to the API. Probably want to remove that.